packages/mobile/src/backup/BackupPhrase.tsx

```diff
@@ -1,12 +1,14 @@
 import Button, { BtnTypes } from '@celo/react-components/components/Button'
+import KeyboardAwareScrollView from '@celo/react-components/components/KeyboardAwareScrollView'
```

Comment:
Once this PR is merged, please post a summary in eng-wallet about the right way to use the new KeyboardAwareScrollView, KeyboardSpacer, and SafeAreaView :)

Reply:
Good idea!